lib/web_ui/lib/src/ui/compositing.dart

```diff
@@ -128,7 +128,7 @@ abstract class SceneBuilder {
   OffsetEngineLayer pushOffset(
```

Comment:
Here and below, do these return types need annotations? Or do only the non-abstract subclasses need the annotations?

Reply:
These should be annotated. Abstract methods don't provide any signal to the tool to decide whether return type can or cannot be null. Done.